Adding new function to hello_world.py

Ex-tela opened the pull request.

Ex-tela committed:
Add files via upload

mvallebr commented on hello_world.py:
```diff
@@ -1,14 +1,25 @@
+    elif lower_choice == 'no':
+        return 'Ok, then. Bye!'
+    else:
+        return 'Your answer was supposed to equal \'yes\' or \'no\'. Try again.'
```
you could keep asking until the answer is either yes or no

mvallebr commented on hello_world.html:
```diff
@@ -1,8 +1,4 @@
 <html>
-<head>
```
why are you removing those?
was this intended?

Ex-tela committed:
Fixing mistake (check additional description)

I'd like to know how to use the HTML indentation, it keeps appearing when I type, but I really don't understand. Should I change the code?

Ex-tela committed:
Changing exceptional responses to be more incisive

check the 'else' in the 2 first functions to see changes

Ex-tela committed:
Merge branch 'main' of github.com:Ex-tela/hello-world into truth_teller_branch

mvallebr reviewed (approved):
LGTM
